### ISSUE TITLE
`lm_robust`'s CIs when `alpha` is specified

### ISSUE THREAD

**Issue by K-Maehashi** [DeclareDesign/estimatr#387]
Thank for this great package! I have a quick question on `lm_robust()`'s output. When I specify `alpha`, `lm_robust()` and `confint()` returns different CIs. Is this an expected behavior?

```r
> library(fabricatr) 
    dat <- fabricate( 
      N = 40, 
      y = rpois(N, lambda = 4), 
      x = rnorm(N), 
      z = rbinom(N, 1, prob = 0.4) 
    )  

> lm_robust(y ~ x + z, data = dat, alpha = 0.2)                                                                                                                                                                                                                                                 
              Estimate Std. Error    t value     Pr(>|t|)   CI Lower  CI Upper DF
(Intercept)  4.1041776  0.3688940 11.1256273 2.316785e-13  3.3567273 4.8516279 37
x           -0.3369374  0.2630956 -1.2806654 2.082812e-01 -0.8700196 0.1961449 37
z           -0.5124764  0.6611455 -0.7751341 4.431864e-01 -1.8520844 0.8271316 37

> lm_robust(y ~ x + z, data = dat, alpha = 0.2) %>% confint()                                                                                                                                                                                                                                   
                  10 %        90 %
(Intercept)  3.6228246 4.585530610
x           -0.6802388 0.006364024
z           -1.3751750 0.350222165
```

**Comment by lukesonnet** [comment]
Hmm I suspect that we aren't passing `alpha` correctly through to confint... I'll investigate, thank you!

**Comment by K-Maehashi** [comment]
Thank you for looking into this! My guess is that `confint()` is OK (it uses `alpha = 0.2` in the second example) but there is an issue when the package prints `lm_robust` objects.

**Comment by graemeblair** [comment]
I believe this has been corrected, but not pushed to CRAN. I see on the main branch:

```
> library(estimatr)
> library(fabricatr) 
> dat <- fabricate( 
+     N = 40, 
+     y = rpois(N, lambda = 4), 
+     x = rnorm(N), 
+     z = rbinom(N, 1, prob = 0.4) 
+ )  

> lm_robust(y ~ x + z, data = dat, alpha = 0.2)   
               Estimate Std. Error    t value     Pr(>|t|)   CI Lower  CI Upper DF
(Intercept)  4.12737269  0.2971478 13.8899664 2.972099e-16  3.7396381 4.5151073 37
x           -0.07305454  0.1703089 -0.4289531 6.704447e-01 -0.2952829 0.1491738 37
z           -0.35684415  0.4081904 -0.8742101 3.876431e-01 -0.8894732 0.1757849 37

> lm_robust(y ~ x + z, data = dat, alpha = 0.2) %>% confint()     
                  10 %      90 %
(Intercept)  3.7396381 4.5151073
x           -0.2952829 0.1491738
z           -0.8894732 0.1757849
```

@lukesonnet should we push current main branch or make any other changes?

**Comment by K-Maehashi** [comment]
Thank you! That makes sense, I was using the CRAN version.

**Comment by lukesonnet** [comment]
Right, looks like you solved this in #383.

I think we might as well submit now. I have a few things I want to work on but not slated to land for another month I'd say.

**Comment by graemeblair** [comment]
On CRAN now thanks @K-Maehashi
